userguide/style_transfer/export-coreml.md

```diff
@@ -37,14 +37,42 @@ styleArray?[styleIndex] = 1.0
 Now, you can stylize your images using:
 ```swift
 let mlModel = MyStyleTransferModel()
-let visionModel = try VNCoreMLModel(for: mlModel)
 
-let styleTransfer = VNCoreMLRequest(model: visionModel, completionHandler: { (request, error) in
-        guard let results = request.results else { return }
+let imageSize = CGSize(width: 600, height: 600)
 
-    for case let styleTransferedImage as VNPixelBufferObservation in results {
-        let imageLayer = CALayer()
-        imageLayer.contents = CIImage(cvPixelBuffer: styleTransferedImage.pixelBuffer, options: [:])
+func stylize(img:UIImage, index:Int, image_size: CGSize) -> UIImage {
```

Comment:
Why is this taking a UIImage? There is a lot of image translation here. Can we simplify it? I think the Vision example takes CImage and we could settle on that.